health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/CreateChecklist.js

```
@@ -159,15 +196,18 @@ module = "HCM";
       }
       codes[question.id] = code;
 
+      let moduleChecklist = "hcm-checklist";
+
       let checklistTypeTemp = checklistType.toUpperCase().replace(/ /g, "_");
       let roleTemp = role.toUpperCase().replace(/ /g, "_");
```

Biome

[error] This let declares a variable that is only assigned once.

'roleTemp' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)